lerna.json

```diff
@@ -1,27 +1,15 @@
 {
   "lerna": "2.0.0",
+  "npmClient": "yarn",
+  "useWorkspaces": true,
   "commands": {
-    "bootstrap": {
-      "ignore": [
-        "crna-kitchen-sink",
-        "react-native-vanilla"
-      ]
-    },
     "publish": {
       "ignore": [
         "cra-kitchen-sink",
-        "crna-kitchen-sink",
-        "react-native-vanilla",
-        "vue-example"
+        "vue-kitchen-sink"
       ]
     }
   },
-  "packages": [
```

Comment:
@Hypnosphi why don't we need this anymore?

Reply:
https://github.com/lerna/lerna#--use-workspaces
> If `--use-workspaces` is true then `packages` will be overridden by the value from `package.json/workspaces`.